routers/repo/issue.go

```diff
@@ -223,8 +225,14 @@ func issues(ctx *context.Context, milestoneID int64, isPullOption util.OptionalB
 			ctx.ServerError("GetIsRead", err)
 			return
 		}
+
+		if isPullOption == util.OptionalBoolTrue {
+			commitStatus[issues[i].PullRequest.ID], _ = issues[i].PullRequest.GetLastCommitStatus()
```

Comment:
`commitStatus[issues[i].PullRequest.ID]` maybe `nil`?

Reply:
It might be nil, yes. I've therefore added checks in `templates/repo/issue/list.tmpl` and `templates/user/dashboard/issues.tmpl` for that.

```
  {{if (index $.CommitStatus .ID)}}
```

Would it be better to always return a valid (but possibly empty) `CommitStatus` from GetLastCommitStatus()?
If so, should we skip this, and just call the method directly from the template?